file-alfresco/src/main/java/uk/gov/justice/services/file/alfresco/AlfrescoFileRemover.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.justice.services.file.alfresco;
+
+import static java.lang.String.format;
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON_TYPE;
+import static uk.gov.justice.services.file.alfresco.Headers.headersWithUserId;
+
+import uk.gov.justice.services.common.configuration.GlobalValue;
+import uk.gov.justice.services.file.api.FileOperationException;
+import uk.gov.justice.services.file.api.remover.FileRemover;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.ws.rs.ProcessingException;
+import javax.ws.rs.core.Response;
+
+@ApplicationScoped
+public class AlfrescoFileRemover implements FileRemover {
+
+    @Inject
+    @GlobalValue(key = "alfrescoDeletePath", defaultValue = "/service/slingshot/doclib/action/file/node/workspace/SpacesStore/")
+    String alfrescoDeletePath;
```

Comment:
We have decided to make all injected fields private, anything new should be marked as private